www/index.html

```diff
@@ -21,18 +33,6 @@
         <script type="text/javascript" src="js/jqplot.dateAxisRenderer.min.js"></script>
         <script type="text/javascript" src="js/jqplot.logAxisRenderer.min.js"></script>
 
-        <script type="text/javascript" src="js/i18n.js"></script>
-        <script type="text/javascript" src="js/contrib.js"></script>
-        <script type="text/javascript" src="js/state.js"></script>
-        <script type="text/javascript" src="js/utils.js"></script>
-        <script type="text/javascript" src="js/index.js"></script>
-
-        <script type="text/javascript" src="js/log.js"></script>
-        <script type='text/javascript' src='js/privacy.js'></script>
-        <script type="text/javascript" src="js/results.js"></script>
-        <script type="text/javascript" src="js/settings.js"></script>
-        <script type="text/javascript" src="js/update.js"></script>
-
         <title>Neubot 0.4.16.9 | Web interface</title>
```

Comment:
I assume you moved scripts on the grounds that scripts that use other scripts should come first. Otherwise I cannot tell what your rationale was, since the related commit is not clear with respect to that.

I suggest to research whether the order in which scripts are included matters or not first. Then, if it does not matter, no issues. Otherwise, I'd put jquery first, then jqplot, then our scripts but index.js and finally index.js.